CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 ___Note:__ Yet to be released changes appear here._
 
 * `CHORE`: remove element templates

```

Comment:
You can link https://github.com/bpmn-io/bpmn-js-properties-panel/issues/879 and mark this commit as closing the issue.

Reply:
adjusted the commit > [a0228be](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/938/commits/a0228bedfaec520fed170f1bdb54804c9b802330)